src/components/navBar/index.tsx

```diff
@@ -1,17 +1,66 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
```

Comment:
can we sort the imports here somewhat like in the format where 
the library imports are first 
then components 
then CSS followed by assets